src/controllers/message.js

```diff
@@ -134,6 +134,8 @@ const listPartnerUsers = async (userId, activeUsers) => {
     const messages = await Message.find({
         $or: [{ postedByUser: userId }, { receivedByUser: userId }],
     })
+        .populate('postedByUser', 'username')
+        .populate('receivedByUser', 'username')
```

Comment:
@larasodjati  this will return username but it will return the user Ids objectId, because they are not converted to string till here!

Reply:
I updated the populate method by adding path, model, and which part we want to exclude. I also fixed populate issues on the list partner controller by modifying the partner variable to return the username and status only.

Reply:
Great Job Lara, this just return username and status. I think userId is important for front-end. @jpremo Can you please have a look, if it is all good?

Reply:
@jpremo could you please take a look and give advice about this? Thank you.